<?xml version="1.0"?>

<launch>
  <arg name="port" default="/dev/ttyUSB0" />

  <node name="driver" pkg="create2_driver" type="create2_driver_node" output="screen">
    <param name="port" value="$(arg port)" />
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find create2_driver)/launch/rviz.rviz" required="true" />

<!--   <include file="$(find create2_driver)/launch/teleop_ps3.launch">
  </include> -->

  <node name="controller" pkg="create2_controller" type="create2_controller_node" output="screen">
    <param name="frame" value="base_link" />
  </node>

<!--   <node name="test" pkg="create2_driver" type="test_node" output="screen">
  </node> -->
</launch>
